test/bdd/go.mod

```diff
@@ -7,14 +7,20 @@ module github.com/trustbloc/hub-router/test/bdd
 go 1.15
 
 require (
-	github.com/cenkalti/backoff/v4 v4.0.2
+	github.com/cenkalti/backoff/v4 v4.1.0
+	github.com/containerd/continuity v0.0.0-20200928162600-f2cc35102c2a // indirect
 	github.com/cucumber/godog v0.9.0
 	github.com/fsouza/go-dockerclient v1.6.5
-	github.com/google/uuid v1.1.1
-	github.com/gorilla/mux v1.7.4
-	github.com/hyperledger/aries-framework-go v0.1.5-0.20201009073544-dcb7f47ab8db
+	github.com/google/uuid v1.1.2
+	github.com/hyperledger/aries-framework-go v0.1.5-0.20201020141652-ce50bc910e28
+	github.com/opencontainers/runc v1.0.0-rc9 // indirect
+	github.com/sirupsen/logrus v1.7.0 // indirect
 	github.com/trustbloc/edge-core v0.1.5-0.20200916124536-c32454a16108
 	github.com/trustbloc/hub-router v0.0.0-00010101000000-000000000000
+	golang.org/x/net v0.0.0-20201009032441-dbdefad45b89 // indirect
 )
 
 replace github.com/trustbloc/hub-router => ../..
+
+// https://github.com/ory/dockertest/issues/208#issuecomment-686820414
+replace golang.org/x/sys => golang.org/x/sys v0.0.0-20200826173525-f9321e4c35a6
```

Comment:
why you need replace

Reply:
Without it, the build fails in macOS with below error.

```
../../../../../../pkg/mod/github.com/docker/docker@v1.4.2-0.20191101170500-ac7306503d23/pkg/term/tc.go:13:28: undefined: unix.SYS_IOCTL
../../../../../../pkg/mod/github.com/docker/docker@v1.4.2-0.20191101170500-ac7306503d23/pkg/term/tc.go:18:28: undefined: unix.SYS_IOCTL
../../../../../../pkg/mod/github.com/docker/docker@v1.4.2-0.20191101170500-ac7306503d23/pkg/term/termios_bsd.go:24:31: undefined: unix.SYS_IOCTL
../../../../../../pkg/mod/github.com/docker/docker@v1.4.2-0.20191101170500-ac7306503d23/pkg/term/termios_bsd.go:37:31: undefined: unix.SYS_IOCTL
```

The issue started after the changes in golang sys lib - https://github.com/golang/go/commit/d140c357442471ca0d56615811fb4226e99fadd5#diff-1237fc76bf8f68100f69b31a8ce17732f20ed244a6e0746461a72ffda226ebd6R5. We need to wait for the dependent libraries to update. Till then this is a workaround.